### DISCUSSION TITLE
[ninja] vcpkg_find_acquire_program(NINJA) downloads x64 binary on Arm

### DISCUSSION THREAD

**Discussion by ture-f-munter** [microsoft/vcpkg#22851] [Q&A]
**Describe the bug**
When installing "Qt5-base" on a Jetson TX2 (64bit Tegra ARM), build of the harfbuzz package failed during setup of the Meson tool. The underlying problem is, that vcpkg_find_acquire_program() in scripts/cmake/vcpkg_find_acquire_program.cmake downloads an x64 binary build of the Ninja buildsystem on all Linux systems and expects it to work.

Ninja (1.8.2) is already installed system-wide using the Ubuntu package. I have set VCPKG_FORCE_SYSTEM_BINARIES=1 before installing the package. VCPKG tries to download Ninja 1.10.1 during the build process.
I suggest that Vcpkg builds Ninja for Linux on non-x64/x86 architectures instead of downloading a binary package. See else() in scripts/cmake/vcpkg_find_acquire_program.cmake:235   (commit# ed54efbb1 2021-03-20 19:56)

**Work around**
Make a symbolic link from downloads/tools/ninja/1.10.1-linux/ninja to /usr/bin/ninja
Works, but is not a good solution.

**Environment**
NVIDIA Jetson TX2 (64bit NVIDIA ARM processor)
- OS: Linux (Ubuntu 18.04LTS)

**To Reproduce**
1. ./vcpkg install harfbuzz:arm64-linux

**Expected behavior**
Would be nice if the Ninja tool works also on Arm.

**Comment by MonocleSecurity** [comment]
I also experience this issue with all the same details, including the TX2!

Temporary fix:

rm downloads/tools/ninja/1.10.1-linux/ninja
ln -s /usr/bin/ninja downloads/tools/ninja/1.10.1-linux/ninja
